src/routes/+layout.svelte

```
@@ -3,13 +3,27 @@
-  import { StatusBadge } from '$lib/modules/client/common';
+  import { Icon } from '$lib/modules/client/common';
+  import { Button, Pill } from '@juspay/svelte-ui-components';
   import { onMount, type Snippet } from 'svelte';
 
   const { children }: { children: Snippet } = $props();
 
   let systemStatus = $state<'degraded' | 'error' | 'healthy' | 'unknown'>('unknown');
 
+  const statusLabels: Record<string, string> = {
+    healthy: 'Online',
+    error: 'Offline',
```

ESLint

[error] Expected "error" to come before "healthy".

(perfectionist/sort-objects)